spec/httpserver/handlers/ok_spec.clj

```diff
@@ -2,10 +2,10 @@
   (:require [speclj.core :refer :all]
             [httpserver.handlers.ok :as ok]))
 
-(import '(com.td.HttpServer HttpResponse))
-
-(describe "OkHandler"
-  (it "returns an HttpRequest with code 200"
-    (let [ok-handler (ok/new-ok-handler)
-      response (.generateResponse ok-handler)]
-      (should= 200 (.responseCode response)))))
+(describe "ok"
+  (it "the map returned has key :code value of 200"
```

Comment:
nitpick: the test description style is a little awkward here; I like how you write them in handler_spec and handler_function_selector_spec, ex. `(it "returns a map with :code 200"`